### DISCUSSION TITLE
Stop tracking certain products

### DISCUSSION THREAD

**Discussion by sushanth88** [jef/streetmerchant#2156] [Help]
Is there any way that I can stop tracking certain products in the list, For example when i run the script it tracks GPU's, CPU's, Game consoles but i only want to track GPU's.

Please suggest.

**Comment by unekgal** [comment]
Edit your .env file to only show the series (brand or model) and stores you want to look at

SHOW_ONLY_BRANDS=
SHOW_ONLY_MODELS=
SHOW_ONLY_SERIES=3070,3080,3090
STORES=amazon,target,bestbuy

**Reply by sushanth88** [reply]
Thank you for the response, I tried filtering out products that are over MSRP by setting the MAX price on each model am tracking but the scan doesn't seem to filter out and yet it sends me alerts for products that are over my max price limits. Can you please help
